server/main.lua

```diff
@@ -393,6 +393,33 @@ QBCore.Commands.Add("paylawyer", Lang:t("commands.paylawyer"), {{name = "id",hel
     end
 end)
 
+QBCore.Commands.Add('fine', 'Fine A Person', {{name = 'id', help = 'Player ID'}, {name = 'amount', help = 'Fine Amount'}}, false, function(source, args)
+    local biller = QBCore.Functions.GetPlayer(source)
+    local Ply = QBCore.Functions.GetPlayer(tonumber(args[1]))
+    local billed = QBCore.Functions.GetPlayer(tonumber(args[1]))
+    local amount = tonumber(args[2])
+    if biller.PlayerData.job.name == "police" then
+        if billed ~= nil then
+            if biller.PlayerData.citizenid ~= billed.PlayerData.citizenid then
+                if amount and amount > 0 then
+                    Ply.Functions.RemoveMoney('bank', amount, "paid-fine")
+                    TriggerClientEvent('QBCore:Notify', source, 'Fine has been issued to offender succesfully', 'success')
+                    TriggerClientEvent('QBCore:Notify', billed.PlayerData.source, 'State Debt Recovery has automatically recovered the fines owed...')
+                    TriggerEvent('qb-bossmenu:server:addAccountMoney', 'police', amount)
+                else
+                    TriggerClientEvent('QBCore:Notify', source, 'Must Be A Valid Amount Above 0', 'error')
+                end
+            else
+                TriggerClientEvent('QBCore:Notify', source, 'You Cannot Fine Yourself', 'error')
+            end
+        else
+            TriggerClientEvent('QBCore:Notify', source, 'Person Not Online', 'error')
+        end
+    else
+        TriggerClientEvent('QBCore:Notify', source, 'No Access', 'error')
+    end
```

Comment:
```suggestion
    if biller.PlayerData.job.name ~= "police" then
        TriggerClientEvent('QBCore:Notify', source, 'No Access', 'error')
    elseif billed == nil then
        TriggerClientEvent('QBCore:Notify', source, 'Person Not Online', 'error')
    elseif biller.PlayerData.citizenid == billed.PlayerData.citizenid then
        TriggerClientEvent('QBCore:Notify', source, 'You Cannot Fine Yourself', 'error')
    elseif not amount or amount <= 0 then
        TriggerClientEvent('QBCore:Notify', source, 'Must Be A Valid Amount Above 0', 'error')
    else
        billed.Functions.RemoveMoney('bank', amount, "paid-fine")
        TriggerClientEvent('QBCore:Notify', source, 'Fine has been issued to offender succesfully', 'success')
        TriggerClientEvent('QBCore:Notify', billed.PlayerData.source, 'State Debt Recovery has automatically recovered the fines owed...')
        TriggerEvent('qb-bossmenu:server:addAccountMoney', 'police', amount)
    end
```
It would be nice if you would validate everything like this to reduce the amount of indentation that is needed.